src/flexbox.rs

```diff
@@ -1311,6 +1311,7 @@ impl Taffy {
                     }
                 }
                 AlignContent::Stretch => 0.0,
+                AlignContent::SpaceEvenly => free_space / (num_lines + 1) as f32,
```

Comment:
This was surprisingly straightforward, although it does make sense that it is :)